content/en/providers/check-in/sp/_index.md

```diff
@@ -223,6 +223,8 @@ attributes that are relevant for user authorisation:
 | Description                                                                                     | SAML Attribute         |
 | ----------------------------------------------------------------------------------------------- | ---------------------- |
 | [VO/group membership/roles of the authenticated user](#vogroup-membership-and-role-information) | `eduPersonEntitlement` |
+| [Capabilities](#capabilities)                                                                   | `eduPersonEntitlement` |
+| [GOCDB roles](#gocdb-roles)                                                                     | `eduPersonEntitlement` |
 | [Level of Assurance (LoA)](#level-of-assurance)                                                 | `eduPersonAssurance`   |
```

Comment:
"Level of Assurance" --> Identity Assurance

Reply:
Done